code/modules/mob/living/simple_animal/abnormality/he/road_home.dm

```diff
@@ -0,0 +1,377 @@
+/mob/living/simple_animal/hostile/abnormality/road_home
+	name = "The Road Home"
+	desc = "An abnormality ressembling a small girl."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "road_home"
+	icon_living = "road_home"
+	maxHealth = 1300
+	health = 1300
+	move_resist = MOVE_FORCE_STRONG //So she can't be yeeted away and delayed indefinitely
+	density = FALSE //This means bodyblocking her isn't an option either.
+	move_to_delay = 10 //She needs to be slow so she doesn't reach home too fast.
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0, WHITE_DAMAGE = 2, BLACK_DAMAGE = 2, PALE_DAMAGE = 2) //Immune to red because catt mentions physical attacks can't hurt her at all.
+	melee_damage_lower = 1
+	melee_damage_upper = 1 //Irrelevant, she does not attack of her own volition
+	speed = 4
+	melee_damage_type = BLACK_DAMAGE
+	armortype = BLACK_DAMAGE
+	can_breach = TRUE
+	threat_level = HE_LEVEL
+	start_qliphoth = 1
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = 45,
```

Comment:
Didn't Road breach in WL due to instinct work? Might want to reduce the success rate on that one.

Reply:
Yeah it's because I made Scaredy Cat instinct and I wasn't sure if I should swap both of their attributes out considering players are already used to Instinct scaredy, but I can change it if we absolutely need to stick to the original material, I'll just need to change records and a few numbers

Reply:
I lowered her instinct work chance and made her counter lower on bad work, which is close enough.